src/main/java/io/javalin/plugin/openapi/OpenApiOptions.kt

```diff
@@ -88,6 +89,16 @@ class OpenApiOptions constructor(val initialConfigurationCreator: InitialConfigu
         }
     }
 
+    fun examples(examples: Map<Class<*>, Map<String, Example>>) {
+        openApiExamples = examples.mapValues { it.value.toMutableMap() }.toMutableMap()
+    }
+
+    inline fun <reified T : Any> addExampleForSchema(name: String, example: Example) = addExampleForSchema(T::class.java, name, example)
+
+    fun <T> addExampleForSchema(clazz: Class<T>, name: String, example: Example) {
+        openApiExamples[clazz] = mutableMapOf(name to example)
```

Comment:
This would override existing examples if a map is already present. You should check that, before overriding an existing map.

Reply:
Hmm, ok. Will fix.

Reply:
Fix, and added to test. Also added a `.clear()` at the end of each test, since the map is global.